hudi-client/hudi-client-common/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
looks like we need this for static mocking to work. Could not get it to work otherwise. 
https://stackoverflow.com/questions/21105403/mocking-static-methods-with-mockito